.github/workflows/django.yml

```diff
@@ -65,6 +65,9 @@ jobs:
           ./scripts/a4-check.sh
       - name: Install Dependencies
         run: |
+          sudo apt update
+          sudo apt install -y gdal-bin
+          sudo apt install libsqlite3-mod-spatialite
```

Comment:
Maybe just for my understanding, when we have above the image as `image: postgis/postgis:16-3.5`, why do we need to install `libsqlite3-mod-spatialite`? Isn't that for `sqlite`?

Reply:
good catch!